androidenhancedvideoplayer/src/main/java/com/profusion/androidenhancedvideoplayer/components/playerOverlay/PlayerIcons.kt

```diff
@@ -95,3 +95,21 @@ fun CheckIcon(modifier: Modifier = Modifier) {
         modifier = modifier
     )
 }
+
+@Composable
+fun ForwardIcon(modifier: Modifier = Modifier) {
+    Image(
+        painter = painterResource(id = R.drawable.ic_forward),
+        contentDescription = stringResource(R.string.controls_forward_description),
+        modifier = modifier.testTag("ForwardIcon")
+    )
+}
+
+@Composable
+fun RewindIcon(modifier: Modifier = Modifier) {
+    Image(
+        painter = painterResource(id = R.drawable.ic_rewind),
+        contentDescription = stringResource(R.string.controls_rewind_description),
+        modifier = modifier.testTag("RewindIcon")
+    )
+}
```

Comment:
You'll need to update it to match the last commit that I merged on the main

```suggestion
fun ForwardIcon(modifier: Modifier = Modifier) {
    Icon(
        painter = painterResource(id = R.drawable.ic_forward),
        tint = Color.White,
        contentDescription = stringResource(R.string.controls_forward_description),
        modifier = modifier.testTag("ForwardIcon")
    )
}

@Composable
fun RewindIcon(modifier: Modifier = Modifier) {
    Icon(
        painter = painterResource(id = R.drawable.ic_rewind),
        tint = Color.White,
        contentDescription = stringResource(R.string.controls_rewind_description),
        modifier = modifier.testTag("RewindIcon")
    )
}
```